app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -128,7 +130,7 @@ class PublicHealthHeader extends React.Component {
         <Modal.Body>
           {this.state.importType === 'saf' && (
             <div className="mb-3">
-              <a href={`${window.location.origin}/Sara%20Alert%20Import%20Format.xlsx`}>Download formatting guidance</a> (Updated 11/30/2021)
+              <a href={`${window.location.origin}/Sara%20Alert%20Import%20Format.xlsx`}>Download formatting guidance</a> (Updated 12/28/2021)
```

Comment:
Can't comment directly on the import guidance file but:
1. Something we do for all excel files is that we make sure the current cell is A1 on all sheets and the current sheet is the first sheet before saving (because excel keeps track of this information), so this is just for consistency
2. The PO team would like the options under the guidance to just be "True|False" (https://mitrecorp.slack.com/archives/C029JM3PFC4/p1638888992148800)

Reply:
@timwongj I made sure that the Excel file is in the state you described and changed the options to "True|False". Thanks!

Reply:
2 more things:

1. I know this was probably like this before already, but would you mind aligning cell D4 to the left to make it consistent with all the other rows in that column
<img width="997" alt="Screen Shot 2021-12-09 at 9 40 37 AM" src="https://user-images.githubusercontent.com/9956561/145417088-99e96213-1a66-4751-8fe8-2e55a6ad6b30.png">

2. Can you save the current cell on A1 on all the excel test fixtures as well?

Reply:
@timwongj See [commit ](https://github.com/SaraAlert/SaraAlert/pull/1168/commits/e09a58b0eddc90c97b63fdb2bd2ad255cab826a8) for 1

See [commit](https://github.com/SaraAlert/SaraAlert/pull/1168/commits/e09a58b0eddc90c97b63fdb2bd2ad255cab826a8) for 2

Reply:
sorry one last thing and I know this wasn't something you touched, but would you mind doing the same for the Sara Alert Format Invalid Headers file?

Reply:
@timwongj Sure, no problem!

Here's the [commit](https://github.com/SaraAlert/SaraAlert/pull/1168/commits/bf6c0d99ec60f8b37c522b4d1a93a7faaa1d9981)

Reply:
thank you!